.circleci/config.yml

```diff
@@ -1,9 +1,27 @@
-version: 2
+version: 2.1
+
+default_steps: &default_steps
+  steps:
+    - checkout
+    - run: npm ci
+    - run: ./run-snapshots.sh
+
 jobs:
-  build:
+  openjdk_8_with_percy:
+    # We've opted this OpenJDK version to be the one that runs and reports Percy's status
     docker:
-      - image: circleci/openjdk:8-node-browsers
-    steps:
-      - checkout
-      - run: npm install
-      - run: ./run-snapshots.sh
+      - image: circleci/openjdk:10-node-browsers
```

Comment:
openjdk:10-node-browsers is openjdk_8?  I wonder if it's worth a comment explaining this to prevent future confusion.

Reply:
Good catch. Fixed with 16d2b4e70aa0fdffe3e672b7e07fd9ad5869dd57

Also found it a bit confusing knowing what the Node version is being used so I investigated what it was and have added a comment above each image here 38072f1349651d16f855bbc472e6f418e31da48f